src/overlays/misc/ovl_kaleido_scope/z_kaleido_collect.c

```diff
@@ -500,59 +535,59 @@ void KaleidoScope_DrawQuestStatus(PlayState* play, GraphicsContext* gfxCtx) {
+                    if (D_8082A150[j] != 255) {
+                        D_8082A150[j] += VREG(50);
+                        if (D_8082A150[j] >= 255) {
+                            D_8082A150[j] = 255;
                         }
                     }
 
-                    pauseCtx->questVtx[sp21A + 0].v.ob[1] = pauseCtx->questVtx[sp21A + 1].v.ob[1] =
-                        VREG(21 + D_8082A124[sp218]);
+                    pauseCtx->questVtx[bufI + 0].v.ob[1] = pauseCtx->questVtx[bufI + 1].v.ob[1] =
+                        VREG(21 + D_8082A124[j]);
 
-                    pauseCtx->questVtx[sp21A + 2].v.ob[1] = pauseCtx->questVtx[sp21A + 3].v.ob[1] =
-                        pauseCtx->questVtx[sp21A + 0].v.ob[1] - 12;
+                    pauseCtx->questVtx[bufI + 2].v.ob[1] = pauseCtx->questVtx[bufI + 3].v.ob[1] =
+                        pauseCtx->questVtx[bufI + 0].v.ob[1] - 12;
 
                     gDPPipeSync(POLY_OPA_DISP++);
 
-                    if (D_8082A124[sp218] == 0) {
+                    if (D_8082A124[j] == 0) {
                         gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, KALEIDO_COLOR_COLLECT_UNK_R, KALEIDO_COLOR_COLLECT_UNK_G,
-                                        KALEIDO_COLOR_COLLECT_UNK_B, D_8082A150[sp218]);
+                                        KALEIDO_COLOR_COLLECT_UNK_B, D_8082A150[j]);
                     } else {
-                        gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 255, 255, 50, D_8082A150[sp218]);
+                        gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 255, 255, 50, D_8082A150[j]);
                     }
 
                     gDPSetEnvColor(POLY_OPA_DISP++, 10, 10, 10, 0);
-                    gSPVertex(POLY_OPA_DISP++, &pauseCtx->questVtx[sp21A], 4, 0);
+                    gSPVertex(POLY_OPA_DISP++, &pauseCtx->questVtx[bufI], 4, 0);
 
-                    gDPLoadTextureBlock(POLY_OPA_DISP++, D_8082A130[D_8082A124[sp218]], G_IM_FMT_IA, G_IM_SIZ_8b, 16,
-                                        16, 0, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK,
-                                        G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD);
+                    gDPLoadTextureBlock(POLY_OPA_DISP++, D_8082A130[D_8082A124[j]], G_IM_FMT_IA, G_IM_SIZ_8b, 16, 16, 0,
+                                        G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK, G_TX_NOMASK,
+                                        G_TX_NOLOD, G_TX_NOLOD);
 
                     gSP1Quadrangle(POLY_OPA_DISP++, 0, 2, 3, 1, 0);
                 }
             }
         } else if (((pauseCtx->mainState >= PAUSE_MAIN_STATE_4) && (pauseCtx->mainState <= PAUSE_MAIN_STATE_6)) ||
                    (pauseCtx->mainState == PAUSE_MAIN_STATE_8)) {
-            sp224 = pauseCtx->ocarinaSongIdx;
-            sp226 = gOcarinaSongButtons[sp224].numButtons;
+            stepGreen = pauseCtx->ocarinaSongIdx;
+            stepRed = gOcarinaSongButtons[stepGreen].numButtons;
```

Comment:
Surely this is fake :copium:

Reply:
maybe add a comment so we can revisit later?